app/models/micropost.rb

```diff
@@ -1,6 +1,9 @@
 class Micropost < ApplicationRecord
   belongs_to :user
-  scope :feed, ->(id){where "user_id = ?",id}
+  scope :feed, lambda{|user|
+    where("user_id IN (:following_ids) OR user_id = :user_id",
```

Comment:
Có cách viết khác:
where "user_id IN (?) OR user_id = ?", user.id, user.following_ids